app/src/main/res/values-v27/styles.xml

```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Deleting this file allowed the theme to actually take effect. For some reason if I put the themes in values-23 with this file still around, I'd get a crash saying it couldn't find the resources. This is odd because I believe the theme should grab elements from any of these folders up until the version of the device? Maybe someone can shed some light on why this may be happening.

Reply:
The answer doesn't seem obvious from your description. In general, I would be careful about referencing @android platform resources like built-in colors, layouts, or styles. It's possible some API levels have these themes and color resources and others don't. I'd have to test it to figure out which line is causing trouble.